templates/Web/Pages/React.List/src/components/ReactList/ListForm.jsx

```diff
@@ -1,20 +1,41 @@
-﻿import React from "react";
```

Comment:
Does it make sense to keep the previous style of `;` at every line end?

Reply:
That makes sense. I just out of habit forget to add those. Thanks for pointing that out.

Reply:
It should be Prettier's job :)